egs/swbd/s5c/local/nnet3/run_tdnn.sh

```diff
@@ -33,6 +34,32 @@ suffix=
 if [ "$speed_perturb" == "true" ]; then
   suffix=_sp
 fi
+
+cnn_opts=()
+if [ "$use_cnn" == "true" ]; then
+# When the cnn option is on, the LDA is replaced by
+# an CNN layer at the front of the network
+# and the recipe uses fbank features as the CNN input
+# As the dimension of the CNN output is usually large, we place
+# a linear layer at the output of CNN for dimension reduction
+# The ivectors are processed through a fully connected affine layer,
+# then concatenated with the CNN bottleneck output and
+# passed to the deeper part of the network.
+# Due to the data compression issue, it is better to convert MFCC
+# to FBANK in the network instead of directly using FBANK features
+# from the storage. This script uses MFCC features as its input
+# and were converted to FBANK features with an inverse of DCT matrix
+# at the first layer of the network.
+
+  cnn_layer="filt_x_dim=3 filt_y_dim=8 filt_x_step=1 filt_y_step=1 num_filters=256 pool_x_size=1 pool_y_size=3 pool_z_size=1 pool_x_step=1 pool_y_step=3 pool_z_step=1"
+  cnn_opts+=(--cnn-layer "$cnn_layer")
+  cnn_bottleneck_dim=256
+  cnn_opts+=(--cnn-bottleneck-dim $cnn_bottleneck_dim)
+  cepstral_lifter="22.0"
```

Comment:
In egs/swbd/s5c/local/nnet3/run_tdnn.sh
> <https://github.com/kaldi-asr/kaldi/pull/526#discussion_r54309055>:
>
> > +# As the dimension of the CNN output is usually large, we place
> > +# a linear layer at the output of CNN for dimension reduction
> > +# The ivectors are processed through a fully connected affine layer,
> > +# then concatenated with the CNN bottleneck output and
> > +# passed to the deeper part of the network.
> > +# Due to the data compression issue, it is better to convert MFCC
> > +# to FBANK in the network instead of directly using FBANK features
> > +# from the storage. This script uses MFCC features as its input
> > +# and were converted to FBANK features with an inverse of DCT matrix
> > +# at the first layer of the network.
> > +
> > +  cnn_layer="filt_x_dim=3 filt_y_dim=8 filt_x_step=1 filt_y_step=1 num_filters=256 pool_x_size=1 pool_y_size=3 pool_z_size=1 pool_x_step=1 pool_y_step=3 pool_z_step=1"
> > +  cnn_opts+=(--cnn-layer "$cnn_layer")
> > +  cnn_bottleneck_dim=256
> > +  cnn_opts+=(--cnn-bottleneck-dim $cnn_bottleneck_dim)
> > +  cepstral_lifter="22.0"
>
> @danpovey <https://github.com/danpovey> Is there a way to figure out if
> there has been cepstral liftering and if yes the cepstral liftering
> coefficient ? I thought that we could determine this from
> conf/mfcc_hires.conf. However as this option could also be specified from
> the command line relying on just the conf file is not safe.
>

Get it from the config file-- I never pass in options to compute-mfcc-feats
except with the config file, I don't think that's even supported at the
script level.
Dan



> Could you let us know what could be the best way to get the cepstral
> lifter coefficient ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/526/files#r54309055>.
>